referrer-policy/generic/common.js

```diff
@@ -0,0 +1,9 @@
+
+// Adds a PRE tag to end of body with propertis of the current page.
+function appendPageInfo(title) {
```

Comment:
This is only for preview purpose if you run the test manually. The test works automated.
Do you suggest I should remove all meta for debugging?